build-system/tasks/presubmit-checks.js

```diff
@@ -341,6 +341,7 @@ var forbiddenTerms = {
       'extensions/amp-access/0.1/amp-access.js',
       'extensions/amp-experiment/0.1/variant.js',
       'extensions/amp-user-notification/0.1/amp-user-notification.js',
+      'dist.3p/current/integration.js',
```

Comment:
sorry if there's mis-understanding, but I don't see it cid used in your change. did you try make a new build and run presubmit again?